codegenerator/cli/templates/static/codegen/src/db/TablesStatic.res

```diff
@@ -42,7 +39,7 @@ module EventSyncState = {
 
   //We need to update values here not delet the rows, since restarting without a row
   //has a different behaviour to restarting with an initialised row with zero values
-  let resetCurrentCurrentSyncStateQuery = `UPDATE ${table.schemaName}.${table.tableName}
+  let resetCurrentCurrentSyncStateQuery = `UPDATE "${Env.Db.publicSchema}"."${table.tableName}"
```

Comment:
Why make this change? I don't mind either way, and I see this file used to access the env anyhow, but it seems counter productive for librifying if we remove thes from the state/config of the table. I personally think ultimately the whole system should run off of a config outside of the environment and the generated registeration can apply the environment variables.

Reply:
Yes. We should move it there at some point. Table shouldn't know anything about postgres implementation.